### ISSUE TITLE
get API ignores footages?

### ISSUE THREAD

**Issue by shrkw** [BenZed/after-effects#26]
Hi, 
get API is really useful. thank you.
I noticed get API cannot fetch FootageItem. Does it work well on your side?

```
        $.writeln(get.footage().count());  // 0
        $.writeln(get(FolderItem).count());  // 7
        $.writeln(get(FolderItem, FootageItem).count());  // 34
        $.writeln(get(FootageItem).count());  // 0
```

it happens on AE 2015, 2017.

**Comment by BenZed** [comment]
Hm, I'll take a look.

**Comment by BenZed** [comment]
Sure enough!
I wonder how I haven't come across this bug before! Thanks very much. 

It has been fixed in: `after-effects 0.4.15`

**Comment by shrkw** [comment]
Great! Thank you!!!
